code/game/objects/items/storage/backpack.dm

```diff
@@ -400,16 +398,16 @@
 	if((hunger > 50) && prob(20))
 		for(var/obj/item/I in contents)
 			if(IS_EDIBLE(I))
-				var/obj/item/food/F = I
-				F.forceMove(user.loc)
+				var/obj/item/food/hunger_breaks = I //If you fed them poundland microwave meals, it probably would kill them
```

Comment:
I guess this is out of scope for this PR, but this code is wrong. It implies everything that `IS_EDIBLE` passes for is `/obj/item/food`, which isn't true.

Reply:
Oh, uh, that's probably quite true I would have fixed it up.